modules/specifying-role-arn-albo-sts.adoc

```diff
@@ -40,7 +40,7 @@ EOF
 +
 [source,terminal]
 ----
-$ cat <<EOF | oc apply -f -
+$ cat <<EOF > | oc apply -f -
```

Comment:
No No here you dont have to change, this right
Change to be done only on this doc `modules/aws-installing-an-aws-load-balancer-operator.adoc`